stream/src/main/java/org/apache/kafka/stream/processor/internals/IngestorImpl.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.clients.processor.internals;
+package org.apache.kafka.stream.processor.internals;
```

Comment:
@guozhangwang Ok to change this later, but assuming that we will change this to use the pause/unpause on the new consumer?